tests/Fixer/PhpUnit/PhpUnitSizeClassFixerTest.php

```diff
@@ -0,0 +1,344 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\PhpUnit;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @internal
+ *
+ * @author Jefersson Nathan <malukenho@phpse.net>
+ *
+ * @covers \PhpCsFixer\Fixer\PhpUnit\PhpUnitInternalClassFixer
+ */
+final class PhpUnitSizeClassFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideFixCases
+     *
+     * @param string     $expected
+     * @param null|mixed $input
+     * @param array      $config
+     */
+    public function testFix($expected, $input = null, $config = [])
+    {
+        $this->fixer->configure($config);
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return [
+            'It does not change normal classes' => [
+                '<?php
+
+class Hello
+{
+}
+',
+            ],
+            'It marks a test class as internal' => [
+                '<?php
+
+/**
+ * @small
+ */
+class Test extends TestCase
+{
+}
+',
+                '<?php
+
+class Test extends TestCase
+{
+}
+',
+            ],
+            'It adds an internal tag to a class that already has a doc block' => [
+                '<?php
+
+/**
+ * @coversNothing
+ *
+ * @small
+ */
+class Test extends TestCase
+{
+}
+',
+                '<?php
+
+/**
+ * @coversNothing
+ */
+class Test extends TestCase
+{
+}
+',
+            ],
+            'It does not change a class that is already internal' => [
+                '<?php
+
+/**
+ * @small
+ */
+class Test extends TestCase
+{
+}
+',
+            ],
+            'It does not change a class that is already internal and has other annotations' => [
+                '<?php
+
+/**
+ * @author malukenho
+ * @coversNothing
+ * @large
+ * @group large
+ */
+class Test extends TestCase
+{
+}
+',
+            ],
+            'It works on other indentation levels' => [
+                '<?php
+
+if (class_exists("Foo\Bar")) {
+    /**
+     * @small
+     */
+    class Test Extends TestCase
+    {
+    }
+}
+',
+                '<?php
+
+if (class_exists("Foo\Bar")) {
+    class Test Extends TestCase
+    {
+    }
+}
+',
+            ],
+            'It works on other indentation levels when the class has other annotations' => [
+                '<?php
+
+if (class_exists("Foo\Bar")) {
+    /**
+     * @author malukenho again
+     *
+     *
+     * @covers \Other\Class
+     *
+     * @small
+     */
```

Comment:
Can a test be added like this but for tab indent?

Reply:
Done